_posts/2019-02-14-customize.md

```diff
@@ -280,18 +280,21 @@ Note: If you have created you own front-end or are using a [third-party plug-in]
 
 ## Change the default presentation
 
-When a new meeting starts, BigBlueButton displays a default presentation.  The file for the default presentation is located in `/var/www/bigbluebutton-default/default.pdf`.  You can replace the contents of this file with your presentation.  Whenever a meeting is created, BigBlueButton will automatically load, convert, and display this presentation for all users.
+When a new meeting starts, BigBlueButton displays a default presentation.  The file for the default presentation is located in `/var/www/bigbluebutton-default/default.pdf`.  You can replace the contents of this file with your presentation.  Whenever a meeting is created, BigBlueButton will automatically load, convert, and display this presentation for all users. On a bigbluebutton update this default may become replaced .
 
 Alternatively, you can change the global default by editing `/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties` and changing the URL for `beans.presentationService.defaultUploadedPresentation`.
 
 ```properties
 # Default Uploaded presentation file
-beans.presentationService.defaultUploadedPresentation=${bigbluebutton.web.serverURL}/default.pdf
+beans.presentationService.defaultUploadedPresentation=${bigbluebutton.web.serverURL}/your-default.pdf
 ```
 
+The file `your-default.pdf` have to become added to the  `/usr/share/nginx/html/` directory. All files added to that directory can be downloaded from your server.
```

Comment:
- two spaces before the `/usr/share/nginx/html/`
- the file actually should be added to `/var/www/bigbluebutton-default/` instead of `/usr/share/nginx/html/`
- `have to become added` bad grammar

Reply:
_the file actually should be added to /var/www/bigbluebutton-default/ instead of /usr/share/nginx/html/_

I can reproduce this problem:

2020/06/09 22:55:03 [error] 2314#2314: *16803 open() "/usr/share/nginx/html/SessionCheatSheet.pdf" failed (2: No such file or directory),

Can you reproduce the case you describe? And if so what could be wrong on my system?

Reply:
sample: http://bbb2.senfcall.de/default.pdf or https://test.bigbluebutton.org/default.pdf
This works on every BBB server, unless you set up your own / handling in your nginx config.